oak/server/rust/oak_glue/src/lib.rs

```diff
@@ -101,17 +101,22 @@ pub unsafe extern "C" fn glue_start(
                 return oak_abi::INVALID_HANDLE;
             }
         };
+        let runtime_config = oak_runtime::RuntimeConfiguration {
```

Comment:
Could `runtime_config` be an input argument to `glue_start`? (Although I don't quite understand when `glue_start ` is called).

Reply:
It could and should, except that we're expecting to flip `main()` over to being in Rust soon, and then the `oak_glue` code will be totally different.  So I'm just hard-coding for now.

(`glue_start` gets called by C++ code over FFI BTW)

Reply:
I see. Thanks.